openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -413,6 +413,17 @@ def get_password_validation_error(password, username=None, email=None):
     """
     return _validate(_validate_password, errors.AccountPasswordInvalid, password, username, email)
 
+def get_confirm_password_validation_error(confirm_password, password):
+    """Get the built-in validation error message for when
+    the confirm password is invalid in some way.
+
+    :param confirm_password: The proposed confirm password (unicode).
+    :param password: The password to match (unicode).
+    :return: Validation error message.
+
+    """
+    return _validate(_validate_confirm_password, errors.AccountPasswordInvalid, confirm_password, password)
```

Comment:
core edx-platform should have the minimum possible line changes. Can you move this to edly app?

Reply:
Same reason as commented above, the functions are closely related